src/components/ebay-menu/index.js

```diff
@@ -75,8 +75,8 @@ function getInitialState(input) {
         isFake,
         label: input.label,
         class: input.class,
-        grow: Boolean(input.grow),
```

Comment:
I'm seeing some non pagination related diffs. I think you need to rebase/merge to get latest from 0.3.0.